src/ORM/Search/SearchContext.php

```diff
@@ -186,6 +186,11 @@ private function prepareQuery($sort, $limit, ?DataList $existingQuery): DataList
             $query = DataList::create($this->modelClass);
         }
 
+        // @todo In CMS 6 change method param from `$limit = false` to `array|string|null $limit = null`
```

Comment:
Could you move this to the TOP of this method, so that it's really obvious that it relates to parameter types

Reply:
Updated